dissect/target/loaders/log.py

```diff
@@ -7,21 +7,22 @@
 
 class LogLoader(Loader):
     LOGS_DIRS = {
-        "EvtxPlugin": "sysvol/windows/system32/winevt/logs",
-        "EvtPlugin": "sysvol/windows/system32/config",
+        ".evtx": "sysvol/windows/system32/winevt/logs",
+        ".evt": "sysvol/windows/system32/config",
     }
 
     @staticmethod
     def detect(path: Path) -> bool:
         return False
 
-    def _map_entry(self, entry: Path) -> str:
-        logs_dir_key = entry.suffix[1:].capitalize()
-        return str(Path(self.LOGS_DIRS[f"{logs_dir_key}Plugin"]).joinpath(entry.name)).lower()
-
     def map(self, target: Target) -> None:
         self.target = target
         vfs = VirtualFilesystem()
-        for entry in self.path.parent.rglob(self.path.name):
-            vfs.map_file(str(self._map_entry(entry)), str(entry))
+        for entry in self.path.parent.glob(self.path.name):
+            ext = entry.suffix.lower()
+            mapping = self.LOGS_DIRS.get(ext, None)
+            if mapping is None:
```

Comment:
```suggestion
            if (mapping := self.LOGS_DIRS.get(ext, None)) is None:
```
Or not a fan of walrus operator? :P

Reply:
Absolutely, big fan! Did not know that people like to call it that. Have to look real close to see it though...